styleguide/components/Navigation/Navigation.js

```diff
@@ -265,6 +265,17 @@ const Navigation = ({ className, onLinkClick }) => (
             Modals
           </NavLink>
         </li>
+        <li className={ css.linkListItem }>
+          <NavLink
+            exact
+            activeClassName={ css.linkActive }
```

Comment:
I question whether modifiers should be included in patterns; they're less a pattern but more a utility - purely for development. Maybe we should create a utility header, similar to [marvel](https://marvelapp.com/styleguide/utility)?